cartridges/app_custom_pd/cartridge/static/default/css/experience/components/commerceAssets/categoryComponent.css

```diff
@@ -0,0 +1,70 @@
+.category-img {
+  -o-object-fit: fill;
+     object-fit: fill;
+}
+
+.tile-container {
+  position: relative;
+}
+
+.cta-link {
+  position: absolute;
+  left: 50%;
+  -webkit-transform: translateX(-50%);
```

Comment:
you don't need -webkit prefix. Webpack sass plugin will generate those.

Reply:
That is a static file and i found a mistake in my gitignore file, these static files shouldn`t be part of the repo